azure-pipelines.yml

```diff
@@ -100,8 +100,29 @@ jobs:
     failOnStderr: true
   - publish: $(Build.StagingDirectory)
     artifact: $(System.JobName)
-- job: release
+- job: test
```

Comment:
I am going to do this when a github action is enable, but you do it right away! :+1: :+1:

Reply:
It was as simple as installing EFL on the CI server and adding 20 lines of code. :)